### PR TITLE
Add documentation for DataFilter step

### PR DESCRIPTION
**Pull request by taysea**
Add documentation related to this PR that added `step` to DataFilter range: https://github.com/grommet/grommet/pull/6942

### CODE REVIEW

File: src/screens/DataFilter.js [hunk 1]
```diff
@@ -56,10 +56,11 @@ const DataFilterPage = () => (
 
         <Property name="range">
           <Description>
-            The range of numeric values. Should not be used with 'options'.
+            The range of numeric values. Should not be used with 'options'. If
+            not specified, the default step is 20.
```
Comment (jcfilben) [inline src/screens/DataFilter.js:60]: Isn't the default step the total range divided by 20?

Comment (taysea) [reply src/screens/DataFilter.js:60]: You're right, let me adjust the wording.

Comment (taysea) [inline src/screens/DataFilter.js:60]:
```suggestion
            not specified, the default step will be calculated as (max - min) / 20.
```

Comment (taysea) [reply src/screens/DataFilter.js]: Adjusted this wording.

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**